run.mac.sh

```diff
@@ -6,6 +6,34 @@ if [ "$osname" != 'Darwin' ]; then
     exit
 fi
 
+mode=""
```

Comment:
I have replaced all occurrences of "$1" with "$mode", assuming that "$1" represented the "--debug" argument when passed. I can change this if required.